Config/Cessna/options.h

```diff
@@ -530,7 +530,7 @@
 // Parameters below are used in the computation of angle of attack and pitch trim.
 // ( INVERTED_NEUTRAL_PITCH is no longer used and should not be used.)
 // If these parameters are not defined, angle of attack and pitch trim will be set to zero.
-// REFERENCE_SPEED                      The nominal speed in meters per second at which the parameters are defined.
+// REFERENCE_SPEED                      The nominal speed in meters per second at which the parameters are defined. (This is only used to predict the stall speed, being half)

```

Comment:
Reference Speed is used by helical turns when predicting expected Angle of Attack, and Expected requried Trim for a given airspeed. Reference Speed is used to created a linear interpolation of prediction for AoA and Trim. Helical Turn will disengage it's algorithms when the air speed is less than half of REFERENCE_SPEEED. T